### ISSUE TITLE
Allow injector vault addr override

### ISSUE THREAD

**Issue by ryanwholey** [hashicorp/vault-helm#200]
I'm looking to use the Vault helm chart to install the sidecar injector as an alternative to applying the charts directly as noted in a [thread in the vault-k8s repo](https://github.com/hashicorp/vault-k8s/issues/15#issuecomment-568225459).

I saw the open pr regarding documentation about how to [apply only the injector](https://github.com/hashicorp/vault-helm/pull/159/files) which get's me almost all the way there aside from updating the vault URL. 

Would you be open to a PR which allows overriding the `AGENT_INJECT_VAULT_ADDR` env var in `injector-deployment.yaml`? 

Something like:
``` yaml
// injector-deployment.yaml

{{- if .Values.injector.vaultAddr }}
- name: AGENT_INJECT_VAULT_ADDR
  value: {{ .Values.injector.vaultAddr }}
{{- else }}
- name: AGENT_INJECT_VAULT_ADDR
  value: {{ include "vault.scheme" . }}://{{ template "vault.fullname" . }}.{{ .Release.Namespace }}.svc:{{ .Values.server.service.port }}
{{- end }}
```

I can see how this could get weird as the chart is intended to bring up the server and is not explicitly intended for deploying the injector, but this change could help clean up a decent amount of our installation logic. Maybe there's a workaround that could play nicely with helm?

Thanks!

**Comment by tvoran** [comment]
Hi @ryanwholey, we implemented something similar in #207, so I'll close this for now. Thanks for the offer!

**Comment by ryanwholey** [comment]
@tvoran 
Great news! Can't wait to try it out
